src/index.ts

```diff
@@ -1,5 +1,6 @@
 export * from './api';
 export * from './config';
 export * from './leverage';
+export * from './deleverage';
```

Comment:
@Jeff-CCH Should collateral-swap be added `infex.ts`? If yes please Ben help fix in this PR.

Reply:
Yes. Please help fix it. Thanks.